packages/site/cms-site/src/blocks/YouTubeVideoBlock.tsx

```diff
@@ -7,13 +7,13 @@ import { withPreview } from "../iframebridge/withPreview";
 import { PropsWithData } from "./PropsWithData";
 
 interface VideoContainerProps {
-    heightInPercent: number;
+    $heightInPercent: number;
```

Comment:
Changed all props in the library to transient props as described in the [migration guide](https://styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v6).